unit_test/entities/test_esdl_object.py

```diff
@@ -265,7 +265,7 @@ def test_multiple_connection(self):
         esdl_object = EsdlObject(pyesdl_from_file(esdl_file_path))
         pipe = esdl_object.get_all_assets_of_type("pipe")[0]
         assets = esdl_object.get_all_assets_of_type("producer")
-        test_list = [(asset.get_id(), asset.get_port_ids()[0]) for asset in assets]
+        test_list = [(asset.get_id(), asset.get_port_ids()[1]) for asset in assets]
```

Comment:
Why get the index?

Reply:
since we return a list of port id, and you need the second entry from this list